src/plugin/__tests__/nodeSpec.spec.ts

```diff
@@ -261,5 +262,35 @@ describe("nodeSpec generation", () => {
         expect(nodeSpec["exampleElement__exampleField"]).toBeTruthy();
       });
     });
+
+    describe("nestedElement", () => {
+      it("should create a NodeSpec for the nestedElement with a value of element+ by default", () => {
+        const nodeSpec = getNodeSpecForField(
+          "exampleElement",
+          "exampleNestedElementField",
+          createNestedElementField({})
+        );
+
+        expect(
+          nodeSpec["exampleElement__exampleNestedElementField"]
+        ).toMatchObject({
+          content: "element+",
+        });
+      });
+
+      it("should create a NodeSpec for the nestedElement using a value defined by the user if specified", () => {
+        const nodeSpec = getNodeSpecForField(
+          "exampleElement",
+          "exampleNestedElementField",
+          createNestedElementField({ content: "element*" })
```

Comment:
Elsewhere this is defined with content as `block*`, is this an inconsistency?

I would have thought that a nested element would contain `element*`, but I suppose the nested elements are also nested in blocks?

I've said nested too many times... 🪆

Reply:
🪆

The top level `block*` content expression is specific to the demo in `prosemirror-elements` - `element+`/`element*` is closer to what we'll see in flexible-content where the top level editor has an `element+` content expression. At the moment there's a bit of a compromise to allow Composer and the Demo to both work.

It would be good to align the demo more closely with what we see in flexible-content so we don't have this discrepancy. I could add that as a follow-up card to follow this one, or we could fix in this PR?

The demo could do with some love in general, e.g. having far fewer elements.

Reply:
That makes sense and sounds good to me. The difference between demo and Composer does confuse me from time to time (the Alt Styles element in this PR for example threw me).